### PR TITLE
fix longdesc resource handling

### PR DESCRIPTION
**Pull request by therealmarv**
fix longdesc resource path according to
https://github.com/openstax/rhaptos.cnxmlutils/issues/172

### CODE REVIEW

**Comment by therealmarv** [comment]
I will add more tests... wip

**Comment by therealmarv** [comment]
ready for review

**Comment by therealmarv** [comment]
will add more tests to satisfy codecov/patch

**Comment by therealmarv** [comment]
@pumazi now ready for review.

File: tests/test_resolvers.py [hunk 1]
```diff
@@ -179,6 +206,49 @@ def test_get_resource_info(self):
...
+        assert len(bad_ref) == 2
+        assert type(bad_ref[0]) == ReferenceNotFound
+        assert type(bad_ref[1]) == ReferenceNotFound
+
+        resolver_media = ReferenceResolver(io.BytesIO(
+            b'<html xmlns="http://www.w3.org/1999/xhtml"><body>'
+            b'<audio src="">'
+            b'</audio></body></html>'),
+            self.faux_plpy, 3)
+        assert resolver_media.fix_media_references() == []
```
Comment (pumazi) [inline tests/test_resolvers.py]: Hmm... I suppose it's not invalid. 👍 Good find.

**Pull request merged**